tools/redcap2linst.php

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env php
+<?php
+
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once "generic_includes.php";
+
+$options = getopt(
+    "f:",
+    [
+        "output-dir:",
+        "file:",
+    ]
+);
+
+$inputFile = $options['f'] ?? $options['file'] ?? '';
+$outputDir = $options['output-dir'] ?? '';
+
+if (empty($inputFile) || empty($outputDir)) {
+    showHelp();
+    exit(1);
+}
+if (!is_dir($outputDir)) {
+    fprintf(STDERR, "Output directory $outputDir doesn't exist.\n");
+    exit(1);
+}
+if (!is_writeable($outputDir)) {
+    fprintf(STDERR, "Output directory $outputDir is not writeable.\n");
+    exit(1);
+}
+
+$fp = fopen($inputFile, "r");
+if ($fp === false) {
+    fprintf(STDERR, "Could not open file $inputFile\n");
+    exit(1);
+}
+
+$headers     = fgetcsv($fp);
+$instruments = [];
+while ($row = fgetcsv($fp)) {
+    $inst = $row[1];
+    if (!isset($instruments[$inst])) {
+        $instruments[$inst] = [];
+    }
+    $linstFormat = toLINST($row[3], $row[0], $row[4], $row[5]);
+    if (!empty($linstFormat)) {
+        $instruments[$inst][] = $linstFormat;
+    }
+}
+fclose($fp);
+
+outputFiles($outputDir, $instruments);
+
+/**
+ * Take a single line from the redcap dictionary and returns the
+ * closest LINST equivalent.
+ *
+ * @param string $redcaptype      The type from REDCap
+ * @param string $redcapfieldname The fieldname from REDCap
+ * @param string $redcaplabel     The label from REDCap
+ * @param string $redcapChoices   The choices columnfrom REDCap
+ *
+ * @return string
+ */
+function toLINST(
+    string $redcaptype,
+    string $redcapfieldname,
+    string $redcaplabel,
+    string $redcapChoices,
+) : string {
+    $label = str_replace($redcaplabel, "\n", "<br /><br />");
```

Comment:
arguments here might be in the wrong place?

Reply:
also, can "\n" just be replaced by one "<br />" instead of two?

Reply:
I think you're right, can you swap the order to test and if it works do a request changes suggestion from the github frontend?

Reply:
@driusan why is a new line replaced by two line breaks instead of just one?